recipes/ntsm/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "ntsm" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/JustinChu/ntsm/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f22a42b71f7a446c45a98518bb6c18613e4fb320d21264903c661a949a7da23
+
+build:
+  skip: true  # [osx or win]
+  number: 0
+  run_exports:
+   - {{ pin_subpackage('ntsm', max_pin="x.x.x") }}
+
+requirements:
+  build:
+   - {{ compiler('cxx') }}
+   - llvm-openmp  # [osx]
+   - make
+  host:
+   - zlib
+   - libgomp
```

```suggestion
   - libgomp # [linux]
```
I see you marked llvm-openmp as OSX only. Shouldn't libgomp be Linux only ?!

---

Oh, that makes sense. Sorry I'm still a bit of a novice when it comes to conda. Out of curiosity, does that mean the OSX-64 test was able to actually install libgomp as it passed?